### ISSUE TITLE
{Freeway-Bug}[Adaptive Cards - Elements Properties] - Screen Reader is not announcing the descriptive information while navigating on more option button.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#9034]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/
OS Version: Dev (OS Build 27695.1000)
Browser Version: Version 130.0.2849.1 (Official build) dev (64-bit)
Screen Reader: Narrator

**Repro Steps:**
1. Open the above URL on the Edge Dev browser.
2. Press tab key and navigate to the Designer tab item and press enter key to select it.
3. Press tab key navigate to the editor section and select any field.
4. Press tab key and navigate to the 'Element Property' button and press enter key to activate it.
5. Navigate to more option button using tab key and observe the issue.

**Actual Result:**
Descriptive name is not defined for the more option button currently it is defined as '...' due to this when screen reader focus lands on it then it is announcing as **'Button data binding.'**

**Observation:**
Same issue is repro for the all more option(...) button present on the screen.

**Expected Result:**
Descriptive name should be defined for the button so that when screen reader focus lands on it then it should announce descriptive information. It should announce something like **'More option for data context button data binding.'**

**Note:**
Same issue is repro with NVDA.

**User Impact:**
It will impact the screen reader user as they will not be able to know the descriptive information about the content due to this they will confuse about the whether focus is on the same control or any different control.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

![Descriptive name is not defined for the more option button  Here it is announcing only as Button then tooltip information](https://github.com/user-attachments/assets/2bfaca31-e9f5-4199-b0db-a5f72d10e27d)

https://github.com/user-attachments/assets/12f57c98-79e8-4536-9051-562c264af3f3

### Card JSON

```JSON
NA
```

**Comment by anna-dingler** [comment]
Hi @vagpt,

This has been fixed but is not released yet. I added the NotReadyToTest label.

Thanks,
Anna

**Comment by vagpt** [comment]
This issue is now working fine, hence closing this issue.

![Image](https://github.com/user-attachments/assets/333d0277-a5f4-46d0-aee9-2fdd26333d27)
